portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,13 @@
  * Adds a random greeting to the page.
  */
 function addFunFact() {
-    const funFacts = ['Jennifer Almost Didn\'t Return For The Final Season',
+
+    //to fetch quote from the servlet and add it to our home page with a heading using ARROW function
+    fetch('/random-quote').then(response => response.text()).then( (quote)=> {
+        document.getElementById('fun-fact-container').innerText= '\n Fun fact about friends 101: \n' + quote;});
+
+    //old code without servlet's use
```

Comment:
Remove the old code. The code is stored in Git's history if you ever need to get it back.